horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/alerting/ChangeDetection.java

```diff
@@ -11,6 +13,7 @@ public class ChangeDetection {
     public String model;
     @NotNull
     @JsonProperty( required = true )
+    @Schema(implementation = JsonNode.class, type = SchemaType.STRING)
     public JsonNode config;
```

Comment:
This property should be an `ObjectNode`,  bugs have occurred in the past when we allow `JsonNode` on the API and objects are pushed into the database as text.  That is what lead to this error: https://github.com/Hyperfoil/Horreum/issues/1073